docs/cn/Running-Alluxio-Locally.md

```diff
@@ -6,8 +6,6 @@ group: Deploying Alluxio
 priority: 1
 ---
 
-# 单机上独立运行Alluxio
```

Comment:
We don't need to place an `h1` in the docs pages as they come from the title of the doc in the `yml` front-matter. So in this case, it was not needed. Same for this header on the docs in `pt` and `en`